.github/scripts/codecheck.py

```diff
@@ -9,14 +9,14 @@
 JSONRPSEE_RE = r'\bjsonrpsee[_a-z0-9]*::'
 
 LICENSE_TEMPLATE = [
-    r'// Copyright \(c\) 202[0-9](-202[0-9])? RBB S\.r\.l',
+    r'// Copyright \(c\) 202[0-9](-202[0-9])? .+',
     r'// opensource@mintlayer\.org',
     r'// SPDX-License-Identifier: MIT',
     r'// Licensed under the MIT License;',
     r'// you may not use this file except in compliance with the License\.',
     r'// You may obtain a copy of the License at',
     r'//',
-    r'//\s+http://spdx\.org/licenses/MIT',
+    r'//\s+https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE',
```

Comment:
Should this regex be tightened up to require one space exactly rather than some whitespace before the URL to enforce more consistency?

Reply:
Yeah it's probably worth it. I'll fix it up